webapp/src/components/content/contentElement.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+import {injectIntl, IntlShape} from 'react-intl'
+
+import {IContentBlock} from '../../blocks/contentBlock'
+import {Utils} from '../../utils'
+
+import {contentRegistry} from './contentRegistry'
+
+// Need to require here to prevent webpack from tree-shaking these away
+// TODO: Update webpack to avoid this
+require('./textElement')
+require('./imageElement')
+require('./dividerElement')
```

Comment:
@jespino, do you happen to know how to add a folder to webpack so we don't need to explicitly import them here?  I found a few references to using chunks, but thought you might know better. More generally, we probably should take a pass at optimizing the webpack settings too.

Reply:
I'll check that, but shouldn't be hard. We can keep it as is now, because we have the TODO there.

Reply:
I rather prefer the `import` syntax